pkg/ingester/flush.go

```diff
@@ -67,6 +67,10 @@ var (
 		Name: "cortex_ingester_dropped_chunks_total",
 		Help: "Total number of chunks dropped from flushing because they have too few samples.",
 	})
+	oldestUnflushedChunkTimestamp = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "cortex_oldest_unflushed_chunk_timestamp",
```

Comment:
Per the naming best practices, it should be `cortex_oldest_unflushed_chunk_time_seconds`: https://prometheus.io/docs/practices/naming/ and a similar metric is `process_start_time_seconds`

Reply:
Prometheus doesn't seem to be consistent either, with several 'timestamp_in_seconds' metrics:

http://demo.robustperception.io:9090/graph?g0.range_input=1h&g0.expr=%7B__name__%3D~%22.%2Btime.*%22%2C%20job%3D%22prometheus%22%7D&g0.tab=1

Reply:
I'm tempted to rename it to `cortex_oldest_unflushed_chunk_timestamp_in_seconds`, would that be fine? Or do you strongly prefer `cortex_oldest_unflushed_chunk_time_seconds`?

Reply:
`cortex_oldest_unflushed_chunk_timestamp_seconds` seems the best. `_in_` is redundant as that is implicit when ending the metric with units in Prometheus.